src/cozmo/robot.py

```diff
@@ -2015,6 +2015,7 @@ def go_to_object(self, target_object, distance_from_object,
 
         Args:
             target_object (:class:`cozmo.objects.ObservableObject`): The destination object.
+                CustomObject instances are not supported.            
```

Comment:
Do we want to add something like "At this time"?
How much time do we think we will have to come back to Cozmo once Vector is launched?